rita_common/test.toml

```diff
@@ -0,0 +1,36 @@
+[payment]
+pay_threshold = "10000000000000"
```

Comment:
this file was moved here from settings, change the path on line 488 to ../../settings/test.toml (or maybe only one ../)  and delete this one to avoid duplication.

Reply:
fixed